template.rb

```diff
@@ -80,7 +80,7 @@
 
 # Install development and test gems
 gem_group :development, :test do
-  gem 'wildland_dev_tools', '~>0.7.0', git: 'git+ssh://git@github.com/wildland/wildland_dev_tools.git'
+  gem 'wildland_dev_tools', '~>0.8.0', git: 'git+ssh://git@github.com/wildland/wildland_dev_tools.git'
```

Comment:
This should probably specify the tag, otherwise this will keep being an issue. See http://bundler.io/git.html for details.